source/MaterialXContrib/MaterialXMaya/README.md

```diff
@@ -2,11 +2,13 @@
 
 ## Description
 
-This module contains code for a sample Maya plug-in which implements Maya surface and texture shading nodes representing MaterialX shaders in Maya's OpenGL viewport (VP2.0). To support this functionality, MaterialX elements are compiled into GLSL code as well as XML shade fragment wrappers required by Maya to integrate the fragments into the final viewport shader.
+This module contains code for a sample Maya plug-in which implements Maya surface and texture shading nodes representing MaterialX nodes in Maya's OpenGL viewport (VP2.0).
+To support this functionality, MaterialX elements are compiled into GLSL code as well as XML shade fragment wrappers required by Maya to integrate the fragments into the final viewport shader.
 
 ## Build
 
-To build this plug-in, the `MATERIALX_BUILD_CONTRIB` build flag must be enabled and the respective `MAYA_DEBUG_DIR` and/or `MAYA_RELEASE_DIR` build directory options set. The minimum required Maya version is Maya 2019.
+To build this plug-in, the `MATERIALX_BUILD_CONTRIB` build flag must be enabled and the respective `MAYA_DEBUG_DIR` and/or `MAYA_RELEASE_DIR` build directory options must be set.
+The minimum required Maya version is Maya 2019.
```

Comment:
Such line breaks make no difference in the rendered Markdown but make lines shorter and diffs clearer in some tools.